electron-app/src/services/rotkehlchen-api.ts

```diff
@@ -59,12 +59,26 @@ export class RotkehlchenApi {
     });
   }
 
+  private validate_status_patch_username(status: number) {
+    return (
+      status == 200 ||
+      status == 300 ||
+      status == 400 ||
+      status == 401 ||
+      status == 409
+    );
+  }
+
   logout(username: string): Promise<boolean> {
     return new Promise<boolean>((resolve, reject) => {
       this.axios
-        .patch<ActionResult<boolean>>(`/users/${username}`, {
-          action: 'logout'
-        })
+        .patch<ActionResult<boolean>>(
```

Comment:
It's a big diff but essentially just added the status code validators in all endpoints. Have a look and tell me if it makes sense @kelsos